docs/concepts/policy/resource-quotas.md

```diff
@@ -84,6 +84,13 @@ define a quota as follows:
 * `gold.storageclass.storage.k8s.io/requests.storage: 500Gi`
 * `bronze.storageclass.storage.k8s.io/requests.storage: 100Gi`
 
+Quota support for local ephemeral storage was added as an alpha feature in release 1.8.
```

Comment:
Could this use a [`feature-state` shortcode](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#feature-state)?